packages/@lwc/engine-core/src/framework/context-provider.ts

```diff
@@ -31,13 +31,13 @@ export function createContextProvider(adapter: WireAdapterConstructor) {
     }
```

Comment:
_not related to this PR_
There is a grammar mistake in the sentence above:
`throw new Error(`Adapter already has a context provider.`);`